lib/BlockedBy.js

```diff
@@ -197,11 +233,24 @@ class BlockedBy extends NodeResque.Plugin {
 
             // If any blocking job is running, then re-enqueue
             if (blockingBuildIds.length > 0) {
-                if (enforceBlockedBySelf && collapse) {
+                // if build is from same event then don't re-enqueue
+                const isSameEvent = await checkMultipleBuildsFromSameEvent(
+                    buildConfig,
+                    blockingBuildIds,
+                    buildId,
+                    this.queueObject.connection.redis
+                );
+
+                if (isSameEvent) {
+                    logger.warn(`Multiple Builds ${buildId} triggered by the same event`);
```

Comment:
this log is redundant, we are already logging in `checkMultipleBuildsFromSameEvent`